### DISCUSSION TITLE
Snare : Uploading Custom Website Issues

### DISCUSSION THREAD

**Discussion by kashmax** [telekom-security/tpotce#1150] [General]
Hi,

Having issues on uploading custom site. So far I did this.

/opt/tpot/docker/tanner/snare/Dockerfile and set --page-dir 10

But every restart it changes to other directories. Also custom pages like test.html will not work because the pages are in meta.json file with hash, tried to put hash for test.html (using md5sum) in the file but it didn't work either.

Thanks

Tpot Version: 22.04

**Comment by notapentester1** [comment]
Hey, did you ever find a solution to this? thx
